### PR TITLE
Working on tips and tricks section

### PR DESCRIPTION
**Pull request by skirmer**
Adding a README and first notebook for the mini tutorials- these will be self contained small notebooks demonstrating a specific task, such a one EDA technique, a deep dive into one Dask feature, or a best practice that users may not know about.

### CODE REVIEW

File: tips-and-tricks/rolling_averages_with_dask.ipynb [hunk 1]
```diff
@@ -0,0 +1,2047 @@
...
+   "source": [
+    "import s3fs\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import dask\n",
+    "import dask.dataframe as dd\n",
+    "from dask.distributed import wait\n",
+    "import datetime\n",
+    "import warnings\n",
+    "warnings.simplefilter(\"ignore\")"
```
Comment (jameslamb) [inline tips-and-tricks/rolling_averages_with_dask.ipynb:120]: I think this warnings-filtering should be dropped. See this explanation from @jsignell : https://github.com/saturncloud/examples/pull/41#pullrequestreview-531827222

File: tips-and-tricks/rolling_averages_with_dask.ipynb [hunk 2]
```diff
@@ -0,0 +1,2047 @@
...
+    "taxi = taxi.loc[t3:t4]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Optional: Fill Missing Dates in Index\n",
+    "\n",
+    "If you don't have gaps in your date field, then you can ignore this entirely! But for many time series use cases, you'll have missing time points, and want to fill those in before calculating a rolling aggregation. The way we'll do that here is to create a brand new complete index, using pandas functions, and then we'll merge that with our starting dataset.\n",
```
Comment (jameslamb) [inline tips-and-tricks/rolling_averages_with_dask.ipynb:518]: hard-won lessons from Uptake :')

**Review by jameslamb — CHANGES_REQUESTED** [review]
I like the idea of this `tip-and-tricks` section, and how you laid out this notebook.

Would it be a lot of extra work to take what you did in the notebook and wrap it into a function people could copy and paste into their own notebooks? Right now this code is broken up line- by-line with explanations for what each line is doing, which is great, but it would still take some effort for me to translate it for my own purposes, especially since some of the lines of code are interactive-only output (like printing the `.head()`  or `.tail()` to show stuff).

I think this would be even more useful if, near the top, you could say something like this:

> If you just want a rolling mean of a column on a Dask DataFrame, copy this function into your notebook and call it like this:

> `...code snippet...`

> Read on to understand how it works

File: tips-and-tricks/rolling_averages_with_dask.ipynb (hunk 1 above)
Comment (skirmer) [reply tips-and-tricks/rolling_averages_with_dask.ipynb:120]: Oh ok- yeah it's just a thing that got copied in from another notebook, I can cut it.

**Comment by skirmer** [comment]
@jameslamb the function idea is good, I could whip that up pretty quick I bet. Let me see what I can do...

File: tips-and-tricks/rolling_averages_with_dask.ipynb (hunk 2 above)
Comment (skirmer) [reply tips-and-tricks/rolling_averages_with_dask.ipynb:516]: LOL yes I know the pain of time series. Do I ever!

**Review by jameslamb — APPROVED** [review]

**Pull request merged**